src/client/driver/index.js

```diff
@@ -19,10 +20,13 @@ var eventUtils        = testCafeCore.eventUtils;
 var modalBackground   = testCafeUI.modalBackground;
 
 
-const COMMAND_EXECUTING_FLAG            = 'testcafe|driver|command-executing-flag';
-const COMMAND_INTERRUPTED_BY_ERROR_FLAG = 'testcafe|driver|command-interrupted-by-error-flag';
-const TEST_DONE_SENT_FLAG               = 'testcafe|driver|test-done-sent-flag';
+const COMMAND_EXECUTING_FLAG = 'testcafe|driver|command-executing-flag';
+const PENDING_PAGE_ERROR     = 'testcafe|driver|pending-page-error';
+const TEST_DONE_SENT_FLAG    = 'testcafe|driver|test-done-sent-flag';
+const PENDING_STATUS         = 'testcafe|driver|pending-status';
 
+var noop = () => {
```

Comment:
we've defined `noop` in `client/runner/automation` to don't move `lodash` to the client scrips and don't increase their size because of this small function